plugins/statistics.koplugin/main.lua

```diff
@@ -284,7 +296,7 @@ function ReaderStatistics:getCurrentStat()
     }
 end
 
-local function generateReadBooksTable(title, dates)
+function ReaderStatistics:generateReadBooksTable(title, dates)
```

Comment:
Why do we need to convert this function to a method of `ReaderStatistics`? I don't see it using `self`?